public/onboarding/checklist.yml

```
@@ -1,28 +1,76 @@
-- title: Découvrir l'écosystème de beta.gouv.fr
+- title: La première semaine - Assimiler les enjeux de la communauté
   items:
-      - id: fiche-membre
-        title: Créer ma fiche membre
-        disabled: true
-        defaultValue: true
-      - id: atelier-onboarding
-        title: |
-            Assister au [1er atelier d'embarquement en visio](https://airtable.com/appebTa6XsY6fDixm/shr5Uaqje8eV9BabU)
-      - id: atelier-onboarding-presentiel
-        title: |
-            Assister à [l'atelier d'embarquement en présentiel](https://espace-membre.incubateur.net/formations?filter=Nouveaux+membres)
-      - id: forum-beta
-        title: Assister à [un forum Beta.gouv.fr](https://doc.incubateur.net/communaute/decouvrir-beta.gouv.fr/la-communaute/les-rituels/seminaire)
-      - id: valeurs-beta
-        title: Découvrir [les valeurs de beta.gouv.fr](https://doc.incubateur.net/communaute/decouvrir-beta.gouv.fr/valeurs#)
-      - id: video-fondamentaux-culture-administrative
-        title: |
-            Regarder la vidéo [Fondamentaux de la culture administrative](https://tube.numerique.gouv.fr/w/wgpehK9PTp5npJQ3Bp9K27)
-- title: Comprendre les standards de beta.gouv.fr
+    - id: fiche-membre
+      title: Compléter ma fiche membre
+      disabled: true
+      defaultValue: true
+    - id: embarquement-visio
+      title: Participer à [L'embarquement en visioconférence obligatoire dans la communauté](embarquement-dans-la-communaute.md)
+    - id: echange-marche-avec-titulaire
+      title: Programmer un échange avec le titulaire de ton marché public si tu es prestataire
+    - id: video-presentation
+      title: Lire l'email de bienvenue et regarder [la vidéo de présentation de beta.gouv.fr](https://www.youtube.com/watch?v=TvFlxDMlbfo)
+    - id: valeurs
```

```suggestion
    - id: valeurs-beta
```

idéalement il faut conserver les mêmes "id" si on veut que la case reste côchée